libtiledbsoma/src/soma/managed_query.cc

```diff
@@ -73,6 +73,27 @@ void ManagedQuery::reset() {
     query_submitted_ = false;
 }
 
+void ManagedQuery::set_layout(ResultOrder result_order) {
```

Comment:
LGTM! For bonus you could have this method return `this` to allow for some fluent-notation at callsites (which may not be useful, just a thought)